### PR TITLE
Feature/td 11214

### PR DESCRIPTION
**Pull request by wangmm0220**

### CODE REVIEW

File: src/query/src/qAggMain.c [hunk 1]
```diff
@@ -5402,6 +5423,194 @@ static void mode_func_finalizer(SQLFunctionCtx *pCtx) {
...
+
+  STailInfo *pInfo = getOutputInfo(pCtx);
+  buildTailStruct(pInfo, pCtx);
+  return true;
+}
+
+static void tail_function(SQLFunctionCtx *pCtx) {
+  STailInfo *pRes = getOutputInfo(pCtx);
+
+//  if (pCtx->stableQuery){
```
Comment (glzhao89) [inline src/query/src/qAggMain.c:5529]: remove commented code?

File: src/query/src/qAggMain.c [hunk 2]
```diff
@@ -5402,6 +5423,194 @@ static void mode_func_finalizer(SQLFunctionCtx *pCtx) {
...
+    for (int32_t i = 0; i < pCtx->size; ++i) {
+      char *data = GET_INPUT_DATA(pCtx, i);
+
+      TSKEY ts = (pCtx->ptsList != NULL)? GET_TS_DATA(pCtx, i):0;
+      do_tail_function_add(pRes, (int32_t)pCtx->param[0].i64, data, ts,
+                           pCtx->inputBytes, &pCtx->tagInfo, NULL, pCtx->currentStage);
+    }
+//  }else{
```
Comment (glzhao89) [inline src/query/src/qAggMain.c:5537]: Same here. Remove commented code?

File: src/query/src/qAggMain.c [hunk 3]
```diff
@@ -5402,6 +5423,194 @@ static void mode_func_finalizer(SQLFunctionCtx *pCtx) {
...
+  if (pCtx->currentStage == MERGE_STAGE) {
+    bytes = pCtx->outputBytes;
+    type = pCtx->outputType;
+    assert(pCtx->inputType == TSDB_DATA_TYPE_BINARY);
+  } else {
+    bytes = pCtx->inputBytes;
+    type = pCtx->inputType;
+  }
+
+//  if(pCtx->stableQuery){
```
Comment (glzhao89) [inline src/query/src/qAggMain.c:5598]: Same here. Remove commented code?

File: src/query/src/qAggMain.c [hunk 4]
```diff
@@ -5402,6 +5423,194 @@ static void mode_func_finalizer(SQLFunctionCtx *pCtx) {
...
+    GET_RES_INFO(pCtx)->numOfRes = pRes->num - pCtx->param[1].i64;
+//  }else{
```
Comment (glzhao89) [inline src/query/src/qAggMain.c:5600]: Same here. Remove commented code?

File: src/query/src/qExecutor.c [hunk 5]
```diff
@@ -6927,7 +6924,7 @@ static void doStateWindowAggImpl(SOperatorInfo* pOperator, SStateWindowOperatorI
 
   bool    masterScan = IS_MASTER_SCAN(pRuntimeEnv);
   int16_t     bytes = pColInfoData->info.bytes;
-  int16_t     type = pColInfoData->info.type;
+//  int16_t     type = pColInfoData->info.type;
```
Comment (glzhao89) [inline src/query/src/qExecutor.c:6895]: Same here. Remove commented code?

File: src/query/src/qExecutor.c [hunk 6]
```diff
@@ -6939,9 +6936,9 @@ static void doStateWindowAggImpl(SOperatorInfo* pOperator, SStateWindowOperatorI
   pInfo->numOfRows = 0;
   for (int32_t j = 0; j < pSDataBlock->info.rows; ++j) {
     char* val = ((char*)pColInfoData->pData) + bytes * j;
-    if (isNull(val, type)) {
-      continue;
-    }
+//    if (isNull(val, type)) {
```
Comment (glzhao89) [inline src/query/src/qExecutor.c:6907]: Same here. Remove commented code?

**Review by shenglian-zhou — APPROVED** [review]

**Pull request merged**